STEERING-COMMITTEE.md

```diff
@@ -139,24 +130,41 @@ Quorum is considered reached when at least half of the members are present.
 
 ## Getting in touch
 
-There are two ways to raise issues to the steering committee for decision:
+There are three ways to raise issues to the steering committee for decision:
 
 1. Emailing the steering committee at
    [steering@knative.team](mailto:steering@knative.team).
    This is a private discussion list to which all members of the committee have
    access.
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
+   
+3. Asking a question on the
+   [#knative-toc-sc-questions](https://cloud-native.slack.com/archives/C04LQCW0C03)
+   channel on [CNCF Slack](https://cloud-native.slack.com/).
 
 ## Composition
 
-The Steering Committee has five (5) seats. Of these five seats, four (4) are elected
-Contributing seats, open to any project contributor, and one (1) is an
-[End User Seat](#end-user-seat) appointed by the Steering Committee.
+The Steering Committee has five (7) seats. These 7 seats are composed as follows:
+
+- Four (4) are elected Contributing Seats, open to any project contributor.
+  Contributing Seats are elected every year, and serve a two-year term.  In the
+  event of a vacancy, the Steering Committee may select a replacement to serve
+  out the remainder of the term.
+
+- One (1) is an [End User Seat](#end-user-seat) appointed by the Steering
+  Committee.  The End User Seat serves until the next elected Steering Committee
+  appoints a replacement in the two months after members are seated, as
+  described below.
+
+- Up to two Selected Seats (2) are appointed by the Steering Committee to
```

Comment:
What does "up to" means exactly ? In the intro, it says, that the SC has 7 members (not: "between 5 and 7 members"). Can these selected seats also be skipped ?

Reply:
Somewhere below:

> The Steering Committee is not required to fill these Selected Seats

But let me make an inline suggestion for updating the "SC has 7 members" part

Reply:
Oops, yes.  This is a 5-to-7 members thing -- if the SC doesn't need outside experts, they aren't required to fill these seats.